Lib/_pyrepl/commands.py

```diff
@@ -32,8 +34,7 @@
 # [completion]
 
 
-# types
-if False:
```

Comment:
@eugenetriguba I think this might be intentional to avoid the overhead of importing the `typing` module.

Reply:
Interesting, I wasn't aware of that, happy to close this out if that is the case 👍

Reply:
It seems like the likely explanation to me based off your comment and a bit of digging around the stdlib (this was the only place I could find in the stdlib that uses TYPE_CHECKING and that's only because it re-defined it, not that it used it from typing: https://github.com/python/cpython/blob/main/Lib/_pyrepl/unix_console.py#L43). If I learn otherwise later, can always re-open it 🤷 Thanks for your thoughts